src/loadshim/startup.d/01-loadapi.lua

```diff
@@ -0,0 +1,18 @@
+package.path = package.path .. ";/modules/?;/modules/?.lua;/modules/?/init.lua"
+
+local old_loadapi = os.loadAPI()
```

Comment:
I think you meant to omit the `()` after `os.loadAPI` here.

Reply:
Whoops, you're right.